server/proxy/pf_client.c

```diff
@@ -483,22 +684,27 @@ static BOOL pf_client_connect(freerdp* instance)
 static DWORD WINAPI pf_client_thread_proc(LPVOID arg)
 {
 	freerdp* instance = (freerdp*)arg;
-	pClientContext* pc = (pClientContext*)instance->context;
-	proxyData* pdata = pc->pdata;
-	DWORD nCount;
+	pClientContext* pc;
+	proxyData* pdata;
+	DWORD nCount = 0;
 	DWORD status;
-	HANDLE handles[65];
+	HANDLE handles[65] = { 0 };
```

Comment:
Should not be `65` but the magic constant for the maximum wait handles (can't remember the name)

Reply:
true